### PR TITLE
Added bug BZ-1857788 as a known issue in the release notes

### PR DESCRIPTION
**Pull request by sjhala-ccs**
BZ-1857788 https://bugzilla.redhat.com/show_bug.cgi?id=1857788 

Added bug to the Known Issues section of the 2.4 release notes. The scope and workaround for this bug were revised this morning in the team leads' meeting.

### CODE REVIEW

File: virt/virt-2-4-release-notes.adoc [hunk 1]
```diff
@@ -122,6 +122,13 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
 //https://bugzilla.redhat.com/show_bug.cig?=id=1959237
 * If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
 
+* Running virtual machines that cannot be live migrated may block an OpenShift cluster upgrade. This includes virtual machines that use hostpath-provisioner storage or SRIOV network interfaces.
```
Comment (jboxman) [inline virt/virt-2-4-release-notes.adoc:125]: s/may/might/
s/OpenShift/{product-title}/
s/SRIOV/SR-IOV/ (for consistency, as I've been using SR-IOV throughout the docs.)

File: virt/virt-2-4-release-notes.adoc [hunk 2]
```diff
@@ -122,6 +122,13 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
...
+As a workaround, reconfigure the virtual machines so that they can be powered off during a cluster upgrade. In the `spec` section of the virtual machine configuration file:
```
Comment (jboxman) [inline virt/virt-2-4-release-notes.adoc:126]: > As a workaround, you can reconfigure

Comment (jboxman) [inline virt/virt-2-4-release-notes.adoc:126]: Is the configuration in a file? Which API is this?

File: virt/virt-2-4-release-notes.adoc [hunk 3]
```diff
@@ -122,6 +122,13 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
...
++
+** Remove the `evictionStrategy: LiveMigrate` field.
+** Set the `runStrategy` to `Always`.
```
Comment (jboxman) [inline virt/virt-2-4-release-notes.adoc:129]: What is `runStrategy`? A field?

> Set the `runStrategy` field to `Always`.

**Review by jboxman — DISMISSED** [review]
Noticed some minor issues.

**Comment by jboxman** [comment]
@vikram-redhat, reviewed.

File: virt/virt-2-4-release-notes.adoc (hunk 2 above)
Comment (sjhala-ccs) [reply virt/virt-2-4-release-notes.adoc]: This is the VirtualMachine object in the KubeVirt API

**Pull request merged**